agreement/types.go

```diff
@@ -68,9 +68,9 @@ const (
 )
```

Comment:
this file should not be changed in this PR

Reply:
ok, reverting this and separate out for another PR then